src/_product-menu.scss

```diff
@@ -26,6 +26,7 @@ $block-product-menu: #{$fd-namespace}-product-menu;
   &__control {
     @include fd-var-color("background-color", fd-color("shell", 2), --fd-color-shell-2);
 
+    background-image: none;
```

Comment:
Can we add this to the button reset instead so we only need to fix it in one place? 
https://github.com/SAP/fundamental-styles/blob/master/src/mixins/_mixins.scss#L184